vulnerabilities/importers/ubuntu.py

```diff
@@ -53,22 +54,22 @@ def __init__(self, *args, **kwargs):
         # we could avoid setting translations, and have it
         # set by default in the OvalParser, but we don't yet know
         # whether all OVAL providers use the same format
-        self.translations = {'less than': '<'}
+        self.translations = {"less than": "<"}
         self.pkg_manager_api = LaunchpadVersionAPI()
 
     def _fetch(self):
-        base_url = 'https://people.canonical.com/~ubuntu-security/oval/'
-        file_name = 'com.ubuntu.{}.cve.oval.xml.bz2'
+        base_url = "https://people.canonical.com/~ubuntu-security/oval/"
```

Comment:
This seems to be duplicated with https://github.com/nexB/vulnerablecode/pull/278/files#diff-616b86ac339cf22cb94532d0b2c631ff58d97935da99e6b9ac8fd95569ab2d50R59